addons/udes_delivery_control/controllers/main.py

```diff
@@ -19,12 +19,21 @@ def _get_picking_info(self, pickings, fields_to_fetch=None):
         Returns:
             list: Returns list of picking dictionary
         """
-        info = ["id", "name", "origin", "state", "u_is_backload", "u_is_unload"]
+        vals = []
+        info = ["id", "name", "origin", "state", "u_is_backload", "u_is_unload", "u_backload_ids"]
         if not fields_to_fetch:
             fields_to_fetch = info
-        return [picking.read(fields_to_fetch)[0] for picking in pickings]
+        for picking in pickings:
+            picking_dict = picking.read(fields_to_fetch)[0]
+            if "u_backload_ids" in fields_to_fetch:
+                picking_dict["u_backload_ids"] = [
+                    {"id": x.id, "supplier_name": x.supplier_id.name}
+                    for x in picking.u_backload_ids
+                ]
+            vals.append(picking_dict)
+        return vals
 
-    @http.route("/api/delivery-control/stock-picking/", type="json", methods=["GET"], auth="user")
+    @http.route("/api/delivery-control/", type="json", methods=["GET"], auth="user")
```

Comment:
Can we also rename the class please? `PickingApi`

Reply:
How does `DeliveryControlApi` sound?